### PR TITLE
fix: forward pgmeta query params

### LINKED ISSUE #1590: "public" schema in studio does not represent postgres "public" schema correctly
**Describe the bug**
In studio version 20230921-d657f29 internal table policies are shown.

**To Reproduce**
1. Update to latest supabase cli
2. `supabase stop` & `supabase start` to fetch latest docker-images
3. Open the studio on `localhost:54323`
4. Open Authentication > Policies

**Expected behavior**
Internal Tables should not be shown under this tab as it is on the hosted version

**Screenshots**
image

**Desktop (please complete the following information):**
 - OS: MacOS 14
 - Browser: Arc
 - Version of CLI: 1.106.1

### PR DESCRIPTION
**Pull request by sweatybridge**
## What kind of change does this PR introduce?

Bug fix https://github.com/supabase/cli/issues/1590

## What is the current behavior?

Please link any relevant issues here.

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

Add any other context or screenshots.

### CODE REVIEW

File: studio/pages/api/pg-meta/[ref]/tables.ts [hunk 1]
```diff
@@ -29,38 +29,29 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
...
+    } else if (value) {
+      query.set(key, value)
+    }
+    return query
+  }, new URLSearchParams())
+
+  let url = `${PG_META_URL}/tables`
+  if (Object.keys(req.query).length > 0) {
+    url += `?${query}`
+  }
```
Comment (kamilogorek) [inline studio/pages/api/pg-meta/[ref]/tables.ts:47]:
```suggestion
  if (query.size > 0) {
    url += `?${query}`
  }
```

**Review by kamilogorek — APPROVED** [review]
Just a nit

File: studio/pages/api/pg-meta/[ref]/tables.ts (hunk 1 above)
Comment (sweatybridge) [reply studio/pages/api/pg-meta/[ref]/tables.ts:47]: .size was added in node 19. I think studio is still on node18? Not entirely sure

Comment (kamilogorek) [reply studio/pages/api/pg-meta/[ref]/tables.ts:47]: eh, yes, it is :( 
https://github.com/supabase/supabase/blob/0ab13a20afc6c484e8b2b2ede20e2ea881c1875b/package.json#L52

**Review by alaister — APPROVED** [review]

**Pull request merged**